### PR TITLE
Add GAZELLE_LOG_LEVEL and GAZELLE_PROGRESS env vars

### PR DESCRIPTION
**Pull request by pcj**

### CODE REVIEW

File: language/debug/config.go [hunk 1]
```diff
@@ -17,6 +18,21 @@ func (dl *debugLang) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Confi
...
+		}
+	}
+	if progress, ok := os.LookupEnv("GAZELLE_PROGRESS"); ok {
+		if progress == "true" || progress == "1" {
+			dc.showTotalElapsedTimeMessages = true
+		}
+	}
+
 	dl.start = time.Now()
 	dl.prev = time.Now()
```
Comment (f0rmiga) [inline language/debug/config.go]:
```suggestion
	now := time.Now()
	dl.start = now
	dl.prev = now
```
This way, they don't drift.

**Comment by pcj** [comment]
cc @f0rmiga

File: language/debug/config.go [hunk 2]
```diff
@@ -17,6 +18,21 @@ func (dl *debugLang) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Confi
 	dc := newDebugConfig()
 	c.Exts[DebugLangName] = dc
 
+	// check for env vars
+	if logLevel, ok := os.LookupEnv("GAZELLE_LOG_LEVEL"); ok {
+		level, err := zerolog.ParseLevel(logLevel)
+		if err != nil {
+			fmt.Printf("warning: bad log_level: %v", err)
```
Comment (f0rmiga) [inline language/debug/config.go]: I'd rather completely fail here.

File: language/debug/config.go [hunk 3]
```diff
@@ -17,6 +18,21 @@ func (dl *debugLang) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Confi
...
+		} else {
+			dc.Logger = dc.Logger.Level(level)
+		}
+	}
+	if progress, ok := os.LookupEnv("GAZELLE_PROGRESS"); ok {
+		if progress == "true" || progress == "1" {
+			dc.showTotalElapsedTimeMessages = true
+		}
```
Comment (f0rmiga) [inline language/debug/config.go]:
```suggestion
		dc.showTotalElapsedTimeMessages = (progress == "true" || progress == "1")
```

**Review by f0rmiga — APPROVED** [review]

**Pull request merged**